pyseir/inference/whitelist.py

```diff
@@ -45,14 +41,11 @@ def generate_whitelist(self, timeseries: MultiRegionTimeseriesDataset) -> pd.Dat
         """
         logging.info("Generating county level whitelist...")
 
-        df_candidates = (
-            timeseries.get_counties()
-            .groupby_region()
-            # Use pandarallel. It doesn't support the `name` attribute so leave FIPS as a regular
-            # column so it can be read in the applied function.
-            .parallel_apply(_whitelist_candidates_per_fips)
-            .reset_index(drop=True)
-        )
+        # Use pandarallel. It doesn't support the `name` attribute so leave FIPS as a regular
+        # column so it can be read in the applied function.
+        df_candidates = pandas_parallel_apply(
```

Comment:
Argh we're going to be dropping the whitelist soon anyways (which will then drop our usage of pandarallel).  Can try to prioritize that so we don't have to work around pandarallel